setup.py

```diff
@@ -61,4 +66,20 @@ def get_cmake_args():
               'jupyterlab-widgets', 'jupyter_nbextensions_configurator', 'nodejs',
               'pythreejs'
           ],
-      })
+      },
+      classifiers=[
+          'Development Status :: 3 - Alpha',
```

Comment:
Is it?

Reply:
What would you put instead? It's not production ready. And for a beta, we would have to have all features in place and only make smaller adjustments and fixes.